internal/pkg/service/stream/storage/level/local/diskwriter/network/router/router_test.go

```diff
@@ -211,7 +211,6 @@ func TestRouter_ShutdownDiskWriterNodeFirst(t *testing.T) {
 {"level":"info","message":"closing sink router","nodeId":"source","component":"sink.router"}
 {"level":"info","message":"closed sink router","nodeId":"source","component":"sink.router"}
 {"level":"info","message":"stopping storage statistics collector","nodeId":"source","component":"statistics.collector"}
-{"level":"debug","message":"sync done","nodeId":"source","component":"statistics.collector"}                                                                
 {"level":"info","message":"storage statistics stopped","nodeId":"source","component":"statistics.collector"}
```

Comment:
Fixed unstable test, ... statics may be synce by the periodical syncer, so there is no record for sync on shutdown.